s2i/s2i.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: s2i
+spec:
+  parameters:
+  - name: BUILDER_IMAGE
+    description: The location of the buildah builder image.
+  - name: BUILDAH_BUILDER_IMAGE
+    description: The location of the buildah builder image.
+    default: docker.io/vdemeester/buildah-builder
+  - name: IMAGE
+    description: The name of the image to push.
```

Comment:
May be _"The name under which to push the resulting built image."_ ?

Reply:
Copied from `buildah` template

Reply:
OK sure